core/templates/dev/head/components/fuzzy_rule_editor_directive.html

```diff
@@ -0,0 +1,2 @@
+<input type="hidden"/>
+<button type="button" class="btn btn-danger" ng-click="removeEntry()">Remove this input</button>
```

Comment:
Hm, why do you have an input type of hidden?

Also, when you say it won't fire: does this mean that, when you do a console.log on line 33 of the JS file (just after $scope.removeEntry()), it doesn't trigger?